appmap/_implementation/configuration.py

```diff
@@ -383,9 +383,7 @@ def __init__(self, *args, **kwargs):
 
 
 def initialize():
+    DetectEnabled.initialize()
     Config().initialize()
     Importer.use_filter(BuiltinFilter)
     Importer.use_filter(ConfigFilter)
-
-
-initialize()
```

Comment:
Why did you remove this call to `initialize`()?

Reply:
[Removing](https://github.com/getappmap/appmap-python/pull/183/commits/edf1f514fb0f96ab25a12ff7ae3619dbf740277c) it lead to not printing 4 WARNINGs unnecessarily.

I've since found a different way to not print these warnings. I can add this call back.

The tests pass without this call.

Reply:
Yes, please it put back, for the sake of consistency with other modules.